web/src/components/icon.jsx

```diff
@@ -1,16 +1,10 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { memo } from 'react';
 
-const IconContainer = styled.svg`
-  height: 24px;
-  width: 24px;
-`;
-
-const path = window.isCordova ? resolvePath('images/material-icon-sprite.svg') : '/images/material-icon-sprite.svg';
+const path = window.isCordova ? resolvePath('images/icon-sprite.svg') : '/images/icon-sprite.svg';
 
 /**
  * To add a new icon to the SVG sprite, open the icons SVG file and copy the the <svg> element.
- * Then paste the the <svg> element and rename it to <symbol> in the file 'material-icon-sprite.svg' and give it a unique id.
+ * Then paste the the <svg> element and rename it to <symbol> in the file 'icon-sprite.svg' and give it a unique id.
  *
```

Comment:
I wanted to move the `weather/wind` icons in here as well, but had some challenges due to the viewBox size - how should one go about resizing icons to match this new unified size? Just dropping it in here didn't work and it was rendered outside of the svg

Reply:
I assume you mean these icons?
![image](https://user-images.githubusercontent.com/30777521/171842102-3e966a7f-5294-4f0d-a3f7-6db018b86a49.png)
They are a little bit special as they are injecting the image/icon as a CSS background-image so I think they need a little extra CSS to work properly. But I'll look into it.

Another option would be trying to switch them to using the `<Icon />` component directly, that should make them render just fine.

Reply:
I'm addressing it in an upcoming PR, so it was mostly to hear your thoughts on how to resize svgs so they match in the sprite :)

Reply:
Oh then I would probably just download the icons from https://fonts.google.com/icons?icon.query=sun&icon.style=Outlined&icon.set=Material+Icons with the correct default size like I did the others and just copy paste the code. I ran the others through https://jakearchibald.github.io/svgomg/ first but that isn't needed (need to be careful with those settings though)

Reply:
Yes, exactly. Will do that for Material icons and then probably replace the ones that are slightly different (wind and "bright mode"/contrast icon) :)

Reply:
For any other icons/svg it should be enough to adjust the viewbox and hight/widht (or remove them as they are not needed with a viewbox) unless they have some inline styling.